anchored-keys/src/main/java/org/infinispan/anchored/impl/AnchoredDistributionInterceptor.java

```diff
@@ -195,9 +197,10 @@ private VisitableCommand replaceWithPutRemoteMetadata(InvocationContext ctx, Dat
          }
 
          Metadata metadata = new RemoteMetadata(keyWriter, null);
-         PutKeyValueCommand copy = new PutKeyValueCommand(key, null, false, false, metadata,
-                                                          command.getSegment(), command.getFlagsBitSet(),
-                                                          command.getCommandInvocationId());
+         PutKeyValueCommand copy = new PutKeyValueCommand(componentRegistry.getCacheByteString(), key, null,
```

Comment:
nitpicking to remove the `componentRegistry` field. 
```suggestion
         PutKeyValueCommand copy = new PutKeyValueCommand(command.getCacheName(), key, null,
```